bar-trender-front-end/src/components/Modals/ModalSearch.js

```diff
@@ -0,0 +1,255 @@
+            </button>
+            <h4 className="title title-up">Filtros</h4>
+          </div>
+          <ModalBody>
+            <Form className="searchbox" onSubmit={this.handleSubmit}>
+
+              <FormGroup>
+                <Input
+                  defaultValue=""
+                  placeholder="Regular"
+                  type="text"
+                  name="nombre"
+                  onChange={this.handleTermChange}
+                ></Input>
+              </FormGroup>
+
+              <FormGroup check>
+
+                <Label check>
+                  <Input type="checkbox"
+                    placeholder="Paulaner"
+                    onChange={this.handleTermChange}
+                    onKeyDown={this.handleEnter}
+                    name="Paulaner"
+                  />
+                  <span className="form-check-sign"></span>
+    Paulaner
+</Label>
+              </FormGroup>
+              <FormGroup check>
+
+                <Label check>
+                  <Input type="checkbox"
+                    placeholder="Triana"
+                    onChange={this.handleTermChange}
+                    onKeyDown={this.handleEnter}
+                    name="Triana"
+                  />
+                  <span className="form-check-sign"></span>
+    Triana
+</Label>
+              </FormGroup>
+              <FormGroup check>
+
+                <Label check>
+                  <Input type="checkbox"
+                    placeholder="Alameda"
+                    onChange={this.handleTermChange}
+                    onKeyDown={this.handleEnter}
+                    name="Alameda"
+                  />
+                  <span className="form-check-sign"></span>
+    Alameda
+</Label>
+              </FormGroup>
+              <FormGroup check>
+
+                <Label check>
+                  <Input type="checkbox"
+                    placeholder="Cruzcampo"
+                    onChange={this.handleTermChange}
+                    onKeyDown={this.handleEnter}
+                    name="Cruzcampo"
+                  />
+                  <span className="form-check-sign"></span>
+    Cruzcampo
+</Label>
+              </FormGroup>
+              <FormGroup>
+                <Col sm="2">
+                  <div className="">
+                    {/* <Button
+                onClick={this.handleSearch}
+                className="btn"
+            >
+                Submit
+         </Button> */}
+                    <Link
+                      onClick={
+                        this.handleSearch,
+                        this.toggle
+                      }
+                      to={{
+                        pathname: '/list',
+                        key: uuid.v4(),
+                        state: [{
+                          Paulaner: this.state['Paulaner'],
+                          Triana: this.state['Triana'],
+                          Alameda: this.state['Alameda'],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7163535)
```suggestion
                          Alameda: this.state["Alameda"],
```